### PR TITLE
Badger std output

### PR DESCRIPTION
**Pull request by cube2222**

### CODE REVIEW

File: output/badger/stdout.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	for range time.Tick(time.Second / 10) {
+		tx := printer.stateStorage.BeginTransaction()
+
+		var buf bytes.Buffer
+
+		records, err := printer.recordsLister.ListRecords(ctx, tx)
+		if err != nil {
+			return errors.Wrap(err, "couldn't list records")
+		}
+		if len(records) > 0 && !records[0].EventTimeField().Empty() {
```
Comment (woojtek) [inline output/badger/stdout.go:47]: maybe newline here

File: output/badger/output.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+	}
+	IDs := octoIDs.AsSlice()
+
+	if !record.IsUndo() {
+		newIDs := octosql.MakeTuple(append(IDs, octosql.MakeString(record.ID().Show())))
+		if err := records.Set(&key, &newIDs); err != nil {
+			return errors.Wrap(err, "couldn't add record ID to output records")
+		}
+	} else {
+		IDs = IDs[:len(IDs)-1]
```
Comment (woojtek) [inline output/badger/output.go:56]: I don't think I understand, how do You know that ID of current record (with undo) is the last one in IDs slice?

File: output/badger/output.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+					execution.WithID(execution.NewRecordID(id.AsString())),
+					execution.WithEventTimeField(o.EventTimeField),
+				),
+			)
+		}
+	}
+	if err := iter.Close(); err != nil {
+		return nil, errors.Wrap(err, "couldn't close iterator")
+	}
+	if err != storage.ErrEndOfIterator {
```
Comment (woojtek) [inline output/badger/output.go:197]: Shouldn't this be directly after for loop? It probably doesn't matter but conceptually first we want to check if the for loop finished without an error and then close the iterator

File: output/badger/output.go (hunk 2 above)
Comment (cube2222) [reply output/badger/output.go:56]: undo's have different ID's than the original records. So we just delete any one of them.

File: output/badger/output.go (hunk 3 above)
Comment (cube2222) [reply output/badger/output.go:197]: sure

**Pull request merged**